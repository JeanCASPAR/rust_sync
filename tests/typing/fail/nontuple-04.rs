use rust_sync::sync;

sync! {
    node oui() = (a, a)
    where
        a : int = 3 + oui();
}

fn main() {}
