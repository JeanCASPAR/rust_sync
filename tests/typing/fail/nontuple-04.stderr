error: tuples don't exist: int * int is not a valid type...
 --> tests/typing/fail/nontuple-04.rs:6:23
  |
6 |         a : int = 3 + oui();
  |                       ^^^^^
